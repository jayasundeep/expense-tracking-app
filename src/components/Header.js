import React from 'react';
import { NavLink } from 'react-router-dom'

const Header = () => (
    <header>
        <h1>Expensify</h1>
        <NavLink to='/' activeClassName='isActive' exact={true} >Go Home</NavLink>
        <NavLink to='/create' activeClassName='isActive' >Add Expense</NavLink>
        <NavLink to='/help' activeClassName='isActive' >Go to Help</NavLink>
    </header>
);

export default Header;