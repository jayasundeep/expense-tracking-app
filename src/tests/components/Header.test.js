import React from 'react';
// import ReactShallowRenderer from 'react-test-renderer/shallow';
import { shallow } from 'enzyme';
import Header from '../../components/Header';

/* test('should render Header component', () => {
    const renderer = new ReactShallowRenderer();
    renderer.render(<Header />);
    expect(renderer.getRenderOutput()).toMatchSnapshot();
    // console.log(renderer.getRenderOutput());
});
 */

test('should render Header component', () => {
    const wrapper = shallow(<Header />);
    expect(wrapper).toMatchSnapshot();
});