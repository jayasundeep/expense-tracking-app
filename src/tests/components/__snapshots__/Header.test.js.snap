// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Header component 1`] = `
<header>
  <h1>
    Expensify
  </h1>
  <NavLink
    activeClassName="isActive"
    exact={true}
    to="/"
  >
    Go Home
  </NavLink>
  <NavLink
    activeClassName="isActive"
    to="/create"
  >
    Add Expense
  </NavLink>
  <NavLink
    activeClassName="isActive"
    to="/edit"
  >
    Edit Expense
  </NavLink>
  <NavLink
    activeClassName="isActive"
    to="/help"
  >
    Go to Help
  </NavLink>
</header>
`;
